synapse/media/_base.py

```diff
@@ -50,6 +50,24 @@
     "text/xml",
 ]
 
+# A list of all content types that are "safe" to be rendered inline in a browser.
```

Comment:
See https://github.com/matrix-org/matrix-react-sdk/blob/a70fcfd0bcf7f8c85986da18001ea11597989a7c/src/utils/blobs.ts for what Element Web will attempt to render inline.